tools/test/toolchains/test_toolchains.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""Test the arm toolchain."""
+
+import os
+import sys
+
+import mock
+
+
+ROOT = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "..", "..", "..")
+)
+sys.path.insert(0, ROOT)
+
+from tools.toolchains.arm import ARM_STD, ARM_MICRO, ARMC6
+from tools.toolchains.gcc import GCC_ARM
+from tools.toolchains.iar import IAR
+
+
+class TestArmToolchain:
```

Comment:
Please use the unittest class for tests

Reply:
[Done](https://github.com/ARMmbed/mbed-os/compare/c72e2ae73a07b2e0f4c388c90f97033cab4693cd..b758988ec8e1ff6101508609587b9a83748cda1a).